setup.py

```diff
@@ -82,6 +82,7 @@
         # transformers 4.10.0 will break the translation model we used here
         "nlp": ["texar-pytorch>=0.1.4"],
         "extractor": ["texar-pytorch>=0.1.4"],
+        "ocr": ["Pillow", "requests", "pytesseract"]
```

Comment:
I think we can rename this extra to `ocr-tutorial`. Since having these packages doesn't really enable OCR, they are just needed for an example/tutorial.